glibc.yaml

```diff
@@ -76,19 +71,24 @@ pipeline:
     runs: |
       cd build
       echo "user-defined-trusted-dirs=/usr/local/lib:/usr/local/lib64:/usr/lib:/usr/lib64:/lib:/lib64" >> configparms.base
-      echo "rtlddir=/lib" >> configparms.base
-      echo "slibdir=/lib" >> configparms.base
+      echo "slibdir=/usr/lib" >> configparms.base
       echo "libdir=/usr/lib" >> configparms.base
       echo "rootsbindir=/usr/bin" >> configparms.base
-
       echo "build-programs=no" > configparms
+      # If x86_64, set to /lib64, /lib for aarch64
+      if [[ ${{build.arch}} == "x86_64" ]]; then
```

Comment:
no to bashism, please use `[ "${{build.arch}} = "x86_64" ]`

Reply:
old habits.... fixed